profile/README.md

```diff
@@ -1,6 +1,60 @@
-[![Code.org](https://raw.githubusercontent.com/code-dot-org/.github/main/profile/fullteam-2023.jpg)](https://code.org)
+[![Code.org](fullteam-2023.jpg)](https://code.org)
 
 Code.org is an education innovation nonprofit dedicated to the vision that every student in every school has the opportunity to learn computer science as part of their core K-12 education. We expand access to computer science in schools, with a focus on increasing participation by young women and students from other underrepresented groups.
 
 Interested in a career with Code.org?
 Learn more at [code.org/about/careers](https://code.org/about/careers).
+
+## Meet the Engineering team at Code.org
+
+![Engineering Team](eng_team.jpg)
+
+This is (most of) us at our in-person get together during the summer of 2023! This crowd is divided into four sub-teams within our engineering team who work together to build and serve the platform for all our students and teachers.
+
+If you are interested in applying for one of our roles, we'd love to hear from you! You can find a list of open positions on our [Careers page](https://code.org/about/careers).
+
+The engineering team at Code.org is divided into four sub-teams that build and support different areas of our platform.
```

Comment:
The four sub-teams bit is repeated on line 12 as well.  I'd suggest removing it above and keeping this one.